.hlint.yaml

```diff
@@ -0,0 +1,76 @@
+# HLint configuration file
```

Comment:
Is this the default file or some additional configurations were added here?
I think it is okay if we use the default one without adding the config file at all  🙂